mlrun/track/__init__.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# flake8: noqa  - this is until we take care of the F401 violations with respect to __all__ & sphinx
+import importlib
+import inspect
+from typing import List, Type
+
+from mlrun.config import config as mlconf
+
+from .tracker import Tracker
+from .tracker_manager import TrackerManager
+
+# Add a tracker to this list for it to be added into the global tracker manager (only if available in the interpreter):
+_TRACKERS = ["mlflow"]
```

Comment:
why capitalized?  is that a convention I missed?

Reply:
module level variables are capitalized from what I'm used to

Reply:
alright. needs to be conventioned better because other places use lower cases.
id default to that

Reply:
what's that? ALL CAPS or lower? I hope CAPS because it will really scratch my brain :(

Reply:
meh. we can take it afterwards. resolves for now